### PR TITLE
OPENEUROPA-1722: Lock php_codesniffer version until our patches are updated.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,4 +41,3 @@
         "sort-packages": true
     }
 }
-
```
Comment (drupol) [inline composer.json]: Unrelated changes.

Comment (brummbar) [reply composer.json]: Double space in composer.

Comment (drupol) [reply composer.json]: Good.
